packages/stt-adapters/gcp/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Converts GCP Speech to Text Json to DraftJs
+ * see `sample` folder for example of input and output as well as `example-usage.js`
+ */
+
+import generateEntitiesRanges from '../generate-entities-ranges/index.js';
+
+const NANO_SECOND = 1000000000;
+
+/**
+ * attribute for the sentences object containing the text. eg sentences ={ punct:'helo', ... }
+ *  or eg sentences ={ text:'hello', ... }
+ * @param sentences
+ */
+export const getBestAlternativeSentence = sentences => {
+  if (sentences.alternatives.length === 0) {
+    return sentences[0];
+  }
+
+  const sentenceWithHighestConfidence = sentences.alternatives.reduce(function(
+    prev,
+    current
+  ) {
+    return parseFloat(prev.confidence) > parseFloat(current.confidence)
+      ? prev
+      : current;
+  });
+
+  return sentenceWithHighestConfidence;
+};
+
+export const trimLeadingAndTailingWhiteSpace = text => {
+  return text.trim();
+};
+
+/**
+ * GCP does not provide a nanosecond attribute if the word starts at 0 nanosecond
+ * @param startSecond
+ * @param nanoSecond
+ * @returns {number}
+ */
+const computeTimeInSeconds = (startSecond, nanoSecond) => {
```

Comment:
more of a question, I haven't seen the spec of the google STT schema, in a word like 

```json
 "startTime": {
                "seconds": "24",
                "nanos": 600000000
              },
```
I am assuming `seconds` and `nanos` need to be combined to get, in this case, the `startTime`?

Reply:
Yea @pietrop , GCP is providing seconds in the following break down , and it the second starts at exactly 24 then it return nothing for nanos attribute. The method is to handle this workflow and to compute the exact time.